### PR TITLE
Split out stanford test shards

### PR DESCRIPTION
**Pull request by stvstnfrd**

### CODE REVIEW

File: openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulksettings.py [hunk 1]
```diff
@@ -139,8 +139,7 @@ def utility_bulksettings_handler(request, course_key_string):
     """
 
     course_key = CourseKey.from_string(course_key_string)
-    response_format = request.REQUEST.get('format', 'html')
-
+    response_format = request.POST.get('format') or request.GET.get('format') or 'html'
```
Comment (stvstnfrd) [inline openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulksettings.py]: `request.REQUEST` was deprecated in Django 1.7 and then removed in Django 1.9 [1].

- [1] https://docs.djangoproject.com/en/1.8/ref/request-response/#django.http.HttpRequest.REQUEST

File: openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulkupdate.py [hunk 2]
```diff
@@ -129,11 +129,8 @@ def utility_bulkupdate_handler(request, course_key_string):
     settings
     """
     course_key = CourseKey.from_string(course_key_string)
-    response_format = request.REQUEST.get('format', 'html')
```
Comment (stvstnfrd) [inline openedx/stanford/cms/djangoapps/contentstore/views/utilities/bulkupdate.py]: We weren't even using this...

File: scripts/circle-ci-tests.sh [hunk 3]
```diff
@@ -32,9 +32,12 @@ function run_paver_quality {
...
     shift
-    flags='--cov-args="-p"'
-    # paver test_system -s "${system}" -t "${test_id}" ${flags}
-    DJANGO_SETTINGS_MODULE=${system}.envs.test pytest ${@}
+    random_flag=''
+    if [ "${system}" == "lms" ]; then
+        random_flag="--randomly-dont-reorganize"
+    fi
+    echo "DJANGO_SETTINGS_MODULE=openedx.stanford.${system}.envs.test pytest ${random_flag} ${@}"
+    DJANGO_SETTINGS_MODULE=openedx.stanford.${system}.envs.test pytest ${random_flag} ${@}
```
Comment (stvstnfrd) [inline scripts/circle-ci-tests.sh]: Yeah.... we'll have better luck if we run w/ `stanford` settings...

File: scripts/circle-ci-tests.sh [hunk 4]
```diff
@@ -44,15 +47,12 @@ function get_children() {
         case ${app} in
             # hawthorn
             cms.djangoapps.contentstore) ;&
-            cms.djangoapps.course_creators) ;&
-            cms.lib.xblock) ;&
```
Comment (stvstnfrd) [inline scripts/circle-ci-tests.sh]: These pass now.

File: scripts/circle-ci-tests.sh [hunk 5]
```diff
@@ -44,15 +47,12 @@ function get_children() {
...
             common.djangoapps.static_replace) ;&
             common.djangoapps.student) ;&
             common.djangoapps.third_party_auth) ;&
             common.djangoapps.util) ;&
             lms.djangoapps.courseware) ;&
             lms.djangoapps.instructor) ;&
-            lms.djangoapps.instructor_task) ;&
```
Comment (stvstnfrd) [inline scripts/circle-ci-tests.sh]: This passes now.

File: scripts/circle-ci-tests.sh [hunk 6]
```diff
@@ -66,7 +66,6 @@ function get_children() {
             openedx.stanford.cms.djangoapps.contentstore) ;&
             openedx.stanford.djangoapps.auth_lagunita) ;&
             openedx.stanford.djangoapps.register_cme) ;&
-            openedx.stanford.lms.djangoapps.instructor_task) ;&
```
Comment (stvstnfrd) [inline scripts/circle-ci-tests.sh]: This passes now.

File: scripts/circle-ci-tests.sh [hunk 7]
```diff
@@ -77,9 +76,9 @@ function get_children() {
             lms.djangoapps.branding) ;&
             lms.djangoapps.ccx) ;&
             openedx.core.djangoapps.cors_csrf) ;&
-            openedx.features.course_experience) ;&
+            openedx.features.course_experience)
             # SEG_FAULT
-            lms.djangoapps.mobile_api)
+            # lms.djangoapps.mobile_api)
```
Comment (stvstnfrd) [inline scripts/circle-ci-tests.sh]: This passes now.

File: scripts/circle-ci-tests.sh [hunk 8]
```diff
@@ -92,6 +91,11 @@ function get_children() {
 }
 
 function run_shard_0() {
+    # run stanford-specific quality checks here
+    true
+}
+
+function run_shard_4() {
```
Comment (stvstnfrd) [inline scripts/circle-ci-tests.sh]: Basically split everything out into stanford and non-stanford shards of each quality/lms/cms/common.

Run stanford first, so we know quickly if we broke our own stuff.

File: scripts/circle-ci-tests.sh (hunk 3 above)
Comment (caesar2164) [reply scripts/circle-ci-tests.sh]: Is there no way to make this change without forking as much?

File: scripts/circle-ci-tests.sh (hunk 7 above)
Comment (caesar2164) [reply scripts/circle-ci-tests.sh]: why comment this out rather than just delete the line?

Comment (stvstnfrd) [reply scripts/circle-ci-tests.sh]: Because I wasn't sure if it would break on Circle or not..

File: scripts/circle-ci-tests.sh (hunk 3 above)
Comment (stvstnfrd) [reply scripts/circle-ci-tests.sh]: Errr, rather, this file _may_ still exist, but edx never upgraded to circle2.0;
they aren't using it.

File: scripts/circle-ci-tests.sh (hunk 7 above)
Comment (caesar2164) [reply scripts/circle-ci-tests.sh]: gotcha

File: scripts/circle-ci-tests.sh (hunk 3 above)
Comment (stvstnfrd) [reply scripts/circle-ci-tests.sh]: Also please remember, we've heavily modified this file for years now...

**Pull request merged**